### ISSUE TITLE
macOS and macOS extended binaries are identical

### ISSUE THREAD

**Issue by mdgreenwald** [gohugoio/hugo#9131]
### What version of Hugo are you using (`hugo version`)?

<pre>
0.89.0

</pre>

### Does this issue reproduce with the latest release?

Yes.

```bash
% md5 hugo_0.89.0_macOS-all.tar.gz && md5 hugo_extended_0.89.0_macOS-all.tar.gz
MD5 (hugo_0.89.0_macOS-all.tar.gz)          = bfbdb6fc069c33d165706b4de6102c38
MD5 (hugo_extended_0.89.0_macOS-all.tar.gz) = bfbdb6fc069c33d165706b4de6102c38

% md5 hugo_extended_0.89.0_macOS-all/hugo && md5 hugo_0.89.0_macOS-all/hugo
MD5 (hugo_extended_0.89.0_macOS-all/hugo) = 0b2fc6415db09a88793a3b42b7b524a0
MD5 (hugo_0.89.0_macOS-all/hugo)          = 0b2fc6415db09a88793a3b42b7b524a0
```

**Comment by bep** [comment]
Thanks for the report; I'm pretty sure I checked this when I tested out the fat binary thing, but ...

**Comment by fris-t** [comment]
Thank you for reporting and checking. I ran into SCSS/SASS issue, but pretty sure to have installed extended. I noticed file sizes of binaries are equal (29.3MB). I will wait for an update. Thank youu!!

**Comment by bep** [comment]
I have released 0.89.1 where I have reverted the "Goreleaser MacOS fat binary" update, which I assumes also fixes this issue, but I'll keep this issue open to track the comeback of that feature.
